lua/starfall/libs_sh/hologram.lua

```diff
@@ -500,6 +500,22 @@ function hologram_methods:setCullMode(mode)
 	holo:SetCullMode(mode==1)
 end
 
+--- Set the render group for a hologram.
+-- @shared
+-- @param number|nil group Render group. If unset, the engine will decide the render group based on the entity's materials.
+function hologram_methods:setRenderGroup(group)
+	local holo = getholo(self)
+	checkpermission(instance, holo, "entities.setRenderProperty")
+	
+	if group then
+		checkluatype(group, TYPE_NUMBER)
```

Comment:
Should also check that the group is a valid integer. Could this be an entity method rather than a hologram method?

Reply:
Would need a separate function for it to be an entity method, since rendergroup only exists on the client and the "shared" stuff done here is accomplished with a specific network variable. I would not mind renaming this one and create an Entity:setRenderGroup(), though